migrations/20170211085502-built-in-strategies.js

```diff
@@ -0,0 +1,15 @@
+/* eslint camelcase: "off" */
```

Comment:
Not required here. Required in other migrations files because of auto-mapping to database-column names where underscore is used as word separator.